src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/disabled_form_alert.html.twig

```diff
@@ -25,6 +25,9 @@
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
 {% block content %}
+  <div class="header-toolbar d-print-none">
```

Comment:
I didn't put the div container in the twig extension because this div is supposed to be in the layout already,  its absence is very specific to the product page